agent/bench-scripts/unittests

```diff
@@ -42,6 +42,10 @@ for script in ${_tdir}/../util-scripts/pbench-* ; do
     cp ${script} ${_testopt}/util-scripts/
     let res=res+${?}
 done
+cp -a $_tdir/../../bin/getconf.py $_testopt/common/bin
```

Comment:
Probably better to copy `$_tdir/../..bin/*` just in case we add more things there in the future.